ParisBNF/abb/BNFabb134.xml

```diff
@@ -250,37 +238,39 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                                          <item xml:id="a4">
                                             <locus target="#65ra"/>
-                                            <desc type="Inventory">A short inventory of works or manuscripts,
+
+                                            <desc type="Inventory" xml:lang="gez">A short inventory of works or manuscripts,
                                               crudely written in a recent hand</desc>
+
                                             </item>
 
                                          <item xml:id="a5">
                                             <locus target="#65v"/>
-                                            <desc type="CalendaricNote">A short list of the feasts of Evangelists and Apostles for
+                                            <desc type="CalendaricNote" xml:lang="gez">A short list of the feasts of Evangelists and Apostles for
```

Comment:
the language should go with `q`, not with `desc`, as the content of `desc` is in English